### DISCUSSION TITLE
Just a thank you

### DISCUSSION THREAD

**Discussion by khamusa** [stimulus-use/stimulus-use#89] [General] [answered]
I just wanted to say thank you for developing and supporting this lib. Simple yet so powerful.

Today I had an issue trying to use `eventPrefix` with `useDispatch` and found a bug where using it on multiple controllers on the same context would cause one prefix to override another. I installed the lib for the first time about 2 weeks ago, and I was surprised to see that not only my issue had been fixed precisely **yesterday** :heart:, but there were already many useful changes, such as the debugging functionalities.

Thank you!

**Comment by adrienpoly** [comment, marked as answer]
🙏 Thanks for the feedback @khamusa very happy to hear that the lib is being tested and used.
